cylc/flow/cfgspec/globalcfg.py

```diff
@@ -333,7 +333,8 @@
                 similar interface to ``scp``.
             ''')
             Conf('ssh command',
-                 VDR.V_STRING, 'ssh -oBatchMode=yes -oConnectTimeout=10',
+                 VDR.V_STRING,
+                 'ssh -oBatchMode=yes -oConnectTimeout=10 -oLogLevel=error',
```

Comment:
> Not so sure we want to make `-oLogLevel=error` a default (will ignore all warnings).
> 
> Was this added to handle the setup of MO systems? If so this is probably something we would have to add to our site platforms configuration.

So, this came about because the MOTD when sshing goes to STERR - this was being returned in the rsync error. We discussed this at team meeting, I offered to either just tack it onto the rsync ssh or to add it to the cylc global ssh cmd and the result of the conversation was to put it in the global. I don't think this will be just a MO problem, as it is common for MoTD to go to STDERR. It is very easy change to move it to the rsync if you would prefer.

Reply:
It just feels a bit dodge to filter out warnings on behalf of users, what if they are legitimate. That said since these commands are used non-interactively there is the question of what value these warnings serve in the first place.

We could expect users/admins to configure their platforms to filter out these messages as required since these messages shouldn't actually break anything, they just clutter up the logging?

Reply:
Ok, I have no major preference either way. @dpmatthews has made the decision for it to stay in the global ssh command. I guess we can always change it later if we get cold feet!

Reply:
Global ssh command is now back to the way it was, with no LogLevel.